### PR TITLE
Add core contributors / copyright header update

### PR DESCRIPTION
**Pull request by alexellis**
## Description

- Core contributors are added by name.
- Copyright notice now encourages OpenFaaS Author(s) in place of
OpenFaaS Project to reflect the nature of distributed copyright.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -112,9 +113,33 @@ Varying levels of write access are made available via our project bot [Derek](ht
...
+The community calls are held on Zoom on a regular basis with invitations sent out via email ahead of time.
+
+General format:
+
+- Project updates/briefing
+- Round-table intros/updates
+- Demos of features/new work from community
+- Q&A
+
 **Governance**
```
Comment (rgee0) [inline CONTRIBUTING.md]: Replace `lead` with `led`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -144,16 +169,18 @@ This project is licensed under the MIT License.
...
 ```
-// Copyright (c) OpenFaaS Project 2017. All rights reserved.
+// Copyright (c) OpenFaaS Author(s) 2018. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 ```
 
 ### Sign your work
 
-> Note: all of the commits in your PR/Patch must be signed-off.
+> Note: every commits in your PR or Patch must be signed-off.
```
Comment (rgee0) [inline CONTRIBUTING.md]: Erroneous plural ‘commits’

**Comment by alexellis** [comment]
Thanks @rgee0

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -112,9 +113,33 @@ Varying levels of write access are made available via our project bot [Derek](ht
...
 
+**Who are the Core Contributors?**
+
+The Core Contributor group includes:
+
+- Alex Ellis (@alexellis)
+- Richard Gee (@rgee0)
+- Stefan Prodan (@stefanprodan)
+- Burton Rheutan (@burtonr)
+- LucasRoesler (@LucasRoesler)
```
Comment (burtonr) [inline CONTRIBUTING.md]: need a space for Lucas' name

File: CONTRIBUTING.md (hunk 1 above)
Comment (burtonr) [reply CONTRIBUTING.md]: I believe it to be pronounced as `leed` as in Alex, not `lead` as in "be in the front"

**Comment by alexellis** [comment]
Thanks for the input chaps.

**Pull request merged**